sensor_array/analysis/execute_genetic_algorithm_analysis.py

```diff
@@ -9,32 +9,17 @@
 # --------------------------------------------------
 # ----- Import RASPA Data and yaml File ------------
 # --------------------------------------------------
-# Redefine system arguments
-# sim_data = sys.argv[1]
-# exp_data = sys.argv[2]
-
-# Data Paths
-# 1 bar
-# sim_data = '/Users/brian_day/Github-Repos/Sensor_Array/ALL_MOFS.csv'
-# exp_data = '/Users/brian_day/Github-Repos/Sensor_Array/exp_data_175.csv'
-
-# 5 and 10 bar
-sim_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research Projects/CO2 Sensing/CO2_RASPA_Results/CO2_5bar/CO2_Sensing_Results_New/ALL_MOFS.csv'
-exp_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research Projects/CO2 Sensing/CO2_RASPA_Results/CO2_5bar/CO2_Sensing_Results_New/exp_data_175.csv'
-
-# Import results as dictionary
-sim_results_import = read_data_as_dict(sim_data)
-exp_results_import = read_data_as_dict(exp_data)
-
 # Import yaml file as dictoncary
-filepath = '/Users/brian_day/Github-Repos/Sensor_Array/settings/process_config.yaml'
+filepath = 'config_files/process_config.sample.yaml'
 data = yaml_loader(filepath)
 
 # Redefine key varaibles in yaml file
+sim_data = data['sim_data']
+exp_data = data['exp_data']
```

Comment:
this method seems like a near duplicate of the above. Can they be combined? Can we have one settings load? Check out how I did my settings loading for the htsohm project: https://github.com/paulboone/htsohm/blob/pmaster/htsohm/config.py